src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Workflow\Dumper\GraphvizDumper;
+use Symfony\Component\Workflow\Marking;
+
+/**
+ * A console command for retrieving information about services.
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class WorkflowDumpCommand extends ContainerAwareCommand
+{
+    public function isEnabled()
+    {
+        return $this->getContainer()->has('workflow.registry');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('workflow:dump')
+            ->setDefinition(array(
+                new InputArgument('name', InputArgument::REQUIRED, 'A workflow name'),
+                new InputArgument('marking', InputArgument::IS_ARRAY, 'A marking (a list of place)'),
+            ))
+            ->setDescription('Dump a workflow')
+            ->setHelp(<<<'EOF'
+The <info>%command.name%</info> command dump a dot representation of a workflow.
```

Comment:
`a dot representation of a workflow` is hard to understand unless you know what a `DOT` filetype is or what Graphviz is. What about something like:

```
... dumps the graphical representation of a workflow in DOT format.
```